packages/sources/icons/src/templates/css.hbs

```diff
@@ -4,7 +4,8 @@
 }
 
 [class^="{{prefix}}-"], [class*=" {{prefix}}-"] {
-    display: inline-flex;
+    display: flex;
+    align-items: center;
```

Comment:
Are you sure that this will break nothing?

Reply:
I will apply these changes in a dedicated branch (must priorise this one)